en/option/series/funnel.md

```diff
@@ -1,215 +1,231 @@
-
-## data(Array)
-{{ use: partial-1d-data-desc }}
-### name(string)
-the name of data item.
-### value(number)
-data value.
-### itemStyle(Object)
-{{use:partial-item-style-desc}}
-{{use:partial-item-style(prefix="###")}}
-#### height(string|number)
-Height of this data item. By default, the height is evenly divided for all data items. The height can be set to percentage (e.g.: '10%') or pixel value (e.g.: 20). Please make sure that the total height of all data items is 100%.
-
-### label(Object)
-The label configuration of a single data item.
-{{use:partial-funnel-label(
-    prefix="###",
-    position=true,
-    formatter=false
-)}}
-
-
-### labelLine(Object)
-{{ use: partial-funnel-label-line(prefix='###') }}
-
-### itemStyle(Object)
-{{use:partial-item-style-desc}}
-{{use:partial-item-style(prefix="###")}}
-
-### emphasis(Object)
-
-#### itemStyle(Object)
-{{use:partial-item-style(prefix="####")}}
-
-#### label(Object)
-{{use:partial-funnel-label(
-    prefix="####",
-    position=false,
-    formatter=false
-)}}
-
-#### labelLine(Object)
-{{use:partial-funnel-label-line(prefix="####")}}
-
-
-{{use: partial-tooltip-in-series-data(
-    galleryViewPath=${galleryViewPath}
-)}}
-
-{{use: partial-marker(
-    prefix="#",
-    galleryEditorPath=${galleryEditorPath},
-    seriesType="funnel"
-)}}
-
-{{use:partial-animation(prefix="#")}}
-
-
-{{use: partial-tooltip-in-series(
-    galleryViewPath=${galleryViewPath}
-)}}
-
-
-
-{{ target: partial-funnel-label }}
-#${prefix} show(boolean) = false
-{{ if: ${position} }}
-#${prefix} position(string) = 'outside'
-Label position.
-
-**Options: **
-+ `'left'`
-
-    Left side of funnel chart. The corresponding trapezoid would be related to through [visual guide line](~series-funnel.labelLine).
-
-+ `'right'`
-
-   Right side of funnel chart. The corresponding trapezoid would be related to through [visual guide line](~series-funnel.labelLine).
-
-+ `'inside'`
-
-    Inside the trapezoid of funnel chart.
-
-+ `'insideRight'`
-
-    Right inside the trapezoid of funnel chart.
-
-+ `'insideLeft'`
-
-    Left inside the trapezoid of funnel chart.
-
-+ `'inner'` equals to `'inside'`.
-+ `'center'` equals to `'inside'`.
-
-{{ /if }}
-{{ if: ${formatter} }}
-#${prefix} formatter(string|Function)
-{{ use: partial-1d-data-label-formatter(extra = {
-    percent: {
-        desc: 'percentage',
-        type: 'number'
-    }
-}) }}
-{{ /if }}
-
-{{ use:partial-text-style(prefix=${prefix}) }}
-
-
-{{ target: partial-funnel-label-line }}
-#${prefix} show(boolean)
-Whether to show visual guide line.
-#${prefix} length(number)
-The length of the first part from visual guide line.
-#${prefix} lineStyle(Object)
-{{use:partial-line-style(prefix="#" + ${prefix})}}
-#${prefix} emphasis(Object)
-The style of visual guide line in emphasis status.
-##${prefix} show(boolean)
-Whether to show visual guide line.
-##${prefix} lineStyle(Object)
-{{use:partial-line-style(prefix="##" + ${prefix})}}
-
-
+
+{{target: series-funnel}}
```

Comment:
The diff is quite strange... It seems these not changed lines are also marked diff. Any idea? @cuijian-dexter